docs/guides/cli.md

```diff
@@ -9,16 +9,29 @@ spectral lint petstore.yaml
 Other options include:
 
 ``` text
-  -e, --encoding=encoding      text encoding to use
-  -f, --format=json|stylish    formatter to use for outputting results
-  -h, --help                   show CLI help
-  -o, --output=output          output to a file instead of stdout
-  -q, --quiet                  no logging - output only
-  -r, --ruleset=ruleset        path to a ruleset file (supports remote files)
-  -s, --skip-rule=skip-rule    ignore certain rules if they are causing trouble
-  -v, --verbose                increase verbosity
+  --version                    Show version number                                          [boolean]
+  --help                       Show help                                                    [boolean]
+  --encoding, -e               text encoding to use                        [string] [default: "utf8"]
+  --format, -f                 formatter to use for outputting results  [string] [default: "stylish"]
+  --output, -o                 output to a file instead of stdout                            [string]
+  --ruleset, -r                path/URL to a ruleset file                                    [string]
+  --skip-rule, -s              ignore certain rules if they are causing trouble              [string]
+  --fail-severity, -F          results of this level or above will trigger a failure exit code
+                                [string] [choices: "error", "warn", "info", "hint"] [default: "hint"]
+  --display-only-failures, -D  only output results equal to or greater than --fail-severity
```

Comment:
`greater` sounds a bit unfortunate, given `DiagnosticSeverity.Error` equals 0, Warn 1 etc.
I know what the intention is, but perhaps there is a way to make it more clear, i.e. `more severe` or `higher/lower`?